### ISSUE TITLE
Many of the test result does not move at all

### ISSUE THREAD

**Issue by davidliudev** [kijai/ComfyUI-DynamiCrafterWrapper#12]
Trying to get some test image moving using the shown workflow but only a couple of them are moving... is it the issue from the model itself or the implementation?

**Comment by kijai** [comment]
It is the model, you can't stray too far from the resolutions they are trained at. The prompt affects it a lot, as does cfg value along with it. The fs value (frame_stride) affects it a lot, lower values force more movement. And then it's very random between seeds, just like SVD.

**Comment by Doubiiu** [comment]
Thanks so much for all your support @kijai. And your answer is exactly correct. Additionally, there is a lot of room to improve this model.

**Comment by davidliudev** [comment]
Thanks for the answer. I tried to tweak the params but no luck. 

I eventually tried another similar ext here https://github.com/chaojie/ComfyUI-DynamiCrafter and seems that it is working normally. 

Not sure if my install of this plugin messed up or what.....
And for 1024 all I get is a messy non-sense output.....

I'll try to get back to this later and see if I could find some clue.

Anyway thanks for all the good work so far!
